Monika After Story/game/script-ch30.rpy

```diff
@@ -545,8 +545,15 @@ label spaceroom(start_bg=None,hide_mask=False,hide_monika=False):
     # d25 seasonal
     if persistent._mas_d25_deco_active:
         $ store.mas_d25_event.showD25Visuals()
-    return
 
+    # player bday
+    if persistent._mas_player_bday_in_player_bday_mode:
```

Comment:
so spaceroom stuff should be only code related to visuals.

Also we want to limit the amount of conditions here. We should aim to only check the obvious var, no_decor.

The autoload check should set the no_decor appropraitely.